### DISCUSSION TITLE
Postgres Default Credentials Template

### DISCUSSION THREAD

**Discussion by pshenoy9** [projectdiscovery/nuclei-templates#4572] [Q&A] [answered]
Hi all, I have been trying to write a Postgres default credential rule (generally CVE-1999-0508) but I am not able to find any similar templates or docs around something like postgres. Does anyone have any insight on how a template can be written for this? My first thought is that it needs to be a network template and wouldn't work over HTTP/S. Thank you!

**Comment by olearycrew** [comment, marked as answer]
@pshenoy9 see the `pgsql-detect` template, that is a network template that shows passing data to PG SQL that may help.
